sdks/go/pkg/beam/io/textio/sdf.go

```diff
@@ -37,17 +37,21 @@ func init() {
 
 // ReadSdf is a variation of Read implemented via SplittableDoFn. This should
 // result in increased performance with runners that support splitting.
+//
+// Deprecated: Called directly by Read, use that instead.
 func ReadSdf(s beam.Scope, glob string) beam.PCollection {
-	s = s.Scope("textio.ReadSdf")
+	s = s.Scope("textio.Read")
```

Comment:
Might be overthinking this, but is this (minorly) breaking if anyone is using this for a composite transform or checking it in a test? It might be worth pulling out the rest of this function out into its own helper and then having each caller of ReadSdf set its scope before calling it.

Relatedly, does this overwrite the scope set by `ReadAllSdf`? (probably not worth changing at this point for the same breaking reason, I'm just curious)

Reply:
IIRC it becomes a sub-scope.